makefile

```diff
@@ -93,13 +97,17 @@ phony += cli clis
 
 # start smoke test specific rules
 $(buildDir)/load-smoke-data:scripts/load-smoke-data.go
-	go build -o $@ $<
+	$(gobin) build -o $@ $<
 $(buildDir)/set-var:scripts/set-var.go
-	go build -o $@ $<
+	$(gobin) build -o $@ $<
 $(buildDir)/set-project-var:scripts/set-project-var.go
-	go build -o $@ $<
+	$(gobin) build -o $@ $<
 set-var:$(buildDir)/set-var
 set-project-var:$(buildDir)/set-project-var
+set-smoke-vars:$(buildDir)/.load-smoke-data
```

Comment:
this target should depend on the binary itself. 

also I don't see the where the .load-smoke-data is created